pkg/database/authorized_app_test.go

```diff
@@ -322,3 +323,33 @@ func TestDatabase_PurgeAuthorizedApps(t *testing.T) {
 		}
 	}
 }
+
+func TestAuthorizedApp_Audits(t *testing.T) {
+	t.Parallel()
+
+	db, _ := testDatabaseInstance.NewDatabase(t, nil)
+	realm, err := db.FindRealm(1)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	authorizedApp := &AuthorizedApp{
+		Name: "Appy",
+	}
+	if _, err := realm.CreateAuthorizedApp(db, authorizedApp, SystemTest); err != nil {
+		t.Fatal(err)
+	}
+
+	authorizedApp.Name = "something else"
```

Comment:
We currently don't have an audit for API key fields. Do users tend to delete and recreate there? If there's an update flow we should cover that